WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -663,6 +666,7 @@ private struct ProductsSection: View {
                 ForEach(viewModel.productRows) { productRow in
                     CollapsibleProductCard(viewModel: productRow,
                                            flow: flow,
+                                           isLoading: viewModel.paymentDataViewModel.isLoading,
```

Comment:
Since we refer `viewModel.paymentDataViewModel.isLoading` multiple times through the file, wdyt if we encapsulate it into computed variable here?

```
private var isLoading: Bool {
        viewModel.paymentDataViewModel.isLoading
    }
```
A bit unrelated but I think there's too many objects in this file and could help with some separation, this could be entirely subjective thought 🤔

Reply:
Yes, I agree that's reasonable improvement 👍